yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -234,6 +235,12 @@ void checkResultInternal(@Nonnull Object actual, @Nonnull String queryDescriptio
                         reportTestFailure(diffMessage);
                     }
                 }
+
+                final var plannerMetrics = resultSet.getStruct(6);
+                if (plannerMetrics != null) {
```

Comment:
Should the `plannerMetrics` ever be `null`? I'd think that it should always be non-null except maybe for mixed mode tests where the column wouldn't be defined (and I guess must not be running here because it's happening in the `explain` step)